testing/fakekds.go

```diff
@@ -128,13 +134,12 @@ func (f *FakeKDS) Get(url string) ([]byte, error) {
 // GetKDS returns an HTTPSGetter that can produce the expected certificates for a given URL in the
 // test environment.
 func GetKDS(t testing.TB) trust.HTTPSGetter {
-	// Insert your own KDS cache here.
-	/*
-	   fakeKds := &FakeKDS{Certs: &kpb.Certificates{}}
-	   if err := proto.Unmarshal(internalKdsCache, fakeKds.Certs); err != nil {
-	   	t.Fatalf("could not unmarshal embedded FakeKDS file: %v", err)
-	   }
-	   return fakeKds
-	*/
-	return nil
+	if *testUseKDS {
+		return nil
```

Comment:
Why return nil if *testUseKDS? Is this to be implemented later?

Reply:
Yeah it's a bit awkward. The idea was to let the nil get replaced down the line with the default getter implementation, but that was before I refactored the getters into the trust package. If I tried using SimpleHTTPSGetter, say, then I would have had a circular dependency. Would you prefer I just make this return trust.DefaultHTTPSGetter()?

Reply:
Done.